community/cypher/cypher-compiler-3.0/src/main/scala/org/neo4j/cypher/internal/compiler/v3_0/helpers/BeansAPIRelationshipIterator.scala

```diff
@@ -23,14 +23,13 @@ import org.neo4j.kernel.impl.api.store.RelationshipIterator
 import org.neo4j.kernel.impl.core.RelationshipProxy.RelationshipActions
 import org.neo4j.graphdb.Relationship
 import org.neo4j.kernel.impl.api.RelationshipVisitor
-import org.neo4j.kernel.impl.core.RelationshipProxy
+import org.neo4j.kernel.impl.core.{NodeManager, RelationshipProxy}
```

Comment:
I know we already have these dependencies but it would be fantastic if we could remove dependencies on kernel from the compiler module rather than adding more.